src/game.cpp

```diff
@@ -4909,7 +4909,7 @@ void game::draw_sidebar()
     const oter_id &cur_ter = overmap_buffer.ter(u.global_omt_location());
 
     werase(w_location);
-    mvwprintz(w_location, 0, 0, cur_ter->get_color(), "%s", utf8_truncate( cur_ter->get_name(), 14 ).c_str());
+    mvwprintz(w_location, 0, 0, cur_ter->get_color(), "%s", utf8_truncate( _( cur_ter->get_name().c_str() ), 14 ).c_str());
```

Comment:
The `name` should be translated right after reading (I mean [here](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/overmap.cpp#L412)), not during utilization.